embedded-graphics/src/image/image_bmp.rs

```diff
@@ -246,8 +250,8 @@ mod tests {
     }
 
     macro_rules! test_pattern {
-        ($color_type:ident, $image_data:expr) => {
-            let image: ImageBmp<$color_type> = ImageBmp::new($image_data).unwrap();
+        ($color_type:ident, $image_data:expr) => {{
```

Comment:
I would just keep using `unwrap` here. This way the tests below don't need to deal with `Result`.